geometry_msgs/PointStamped point
bool on_object
string object_name
